### ISSUE TITLE
[Accessibility] Touch Target is not respected.

### ISSUE THREAD

**Issue by andreacantone** [thunderbird/thunderbird-android#5059]
## Description

The size of the elements should be at least 48dp*48dp. Some elements don't respect the size of the touch target.

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app K9 Mail app.
3. Activate the registration from the Accessibility Scanner and use the k9 Mail app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201118_164305](https://user-images.githubusercontent.com/25429624/99553211-2cf96900-29be-11eb-8c0b-07be923efe70.png)

**Comment by wmontwe** [comment]
The account handling has been updated and this is not the case anymore.
